modules/ROOT/pages/authorized-namespaces.adoc

```diff
@@ -72,11 +73,29 @@ subjects:
   name: rtf-agent-sa
   namespace: default
 ----
+
+To use clustered apps in authorized namespaces mode, the following RoleBinding must be created:
```

Comment:
Globally: 
To avoid the passive voice, I'd say:
"...you must create the following `RoleBinding`:"
Should it be RoleBinding or `RoleBinding`?